meta-luneos/recipes-connectivity/ofono/ofono-halium_1.29.bb

```diff
@@ -11,6 +11,7 @@ DEPENDS = "dbus glib-2.0 udev mobile-broadband-provider-info ell"
 COMPATIBLE_MACHINE = "^halium$"
 # This recipe acts as a replacement for the ofono recipe
 PROVIDES += "ofono"
+RPROVIDES:${PN} += "ofono"
```

Comment:
Add the same for ${PN}-tests or better use VIRTUAL-RUNTIME where ofono and ofono-tests are pulled into the image and change VIRTUAL-RUNTIME value in halium.inc to explicitly use ofono-halium and ofono-halium-tests (assuming there are only a few places which add runtime dependency).